src/devices/src/virtio/net/device.rs

```diff
@@ -152,8 +152,8 @@ impl Net {
         let mut config_space = ConfigSpace::default();
         if let Some(mac) = guest_mac {
             config_space.guest_mac = mac;
-            // When this feature isn't available, the driver generates a random MAC address.
```

Comment:
nit grammar: "will generate" or "generates"

Reply:
Am I blind or there is not grammar error?